pkg/apis/submariner.io/v1/types.go

```diff
@@ -81,6 +83,14 @@ type GatewayStatus struct {
 	Connections   []Connection `json:"connections"`
 }
 
+type LatencySpec struct {
+	LastRTT    string `json:"lastRTT"`
+	MinRTT     string `json:"minRTT"`
+	AverageRTT string `json:"averageRTT"`
+	MaxRTT     string `json:"maxRTT"`
+	StdDevRTT  string `json:"stddevRTT"`
```

Comment:
What format/units will the values be in? Should they be a numeric type? I think we should add some documentation, at least at the struct level.

Reply:
@aswinsuryan ^^^

Reply:
I changed it to time.Duration so the unit is now a standard one. Added documentation st the struct level.